src/constants.ts

```diff
@@ -36,4 +36,17 @@ export enum CCLicenseAdaption {
   ALIKE = 'alike',
 }
 
-export const SCREEN_MAX_HEIGHT = window.innerHeight * 0.8;
+export const DEFAULT_LOADER_SIZE = 20;
+
+export const DRAG_COLUMN_WIDTH = 22;
+
+export const HEADER_USERNAME_MAX_WIDTH = 120;
+
+export const PDF_VIEWER_LINK = `${process.env.REACT_APP_GRAASP_ASSETS_URL}/pdf-viewer/web/viewer.html?file=`;
+
+export const GRAASP_ASSETS_PROTOCOL = 'https://';
+
+export const SCREEN_MAX_HEIGHT =
+  typeof window !== 'undefined' ? window.innerHeight * 0.8 : 1000;
+
+export const FAVORITE_COLOR = '#ffc107';
```

Comment:
Woops did I screw up the merge? @spaenleh have those been removed from UI since? The diff tells me that the merge would add them back in